test/e2e/flowaggregator_test.go

```diff
@@ -253,17 +252,47 @@ func checkRecordsForFlows(t *testing.T, data *TestData, srcIP string, dstIP stri
 					}
 				}
 			}
+			// Check the aggregation results in infra tests
+			if !strings.Contains(record, fmt.Sprintf("%s: %s", "destinationPodNamespace", testNamespace)) {
+				t.Errorf("Records does not have correct destinationPodNamespace")
```

Comment:
s/Records does not have/Record does not have

here and in all the error messages below

Reply:
Thanks, addressed.